GetStream/stream-chat-swift
# Add support for custom HTTP headers in `ChatClientConfig`

### 🔗 Issue Links
Resolves https://github.com/GetStream/ios-issues-tracking/issues/568

### 🎯 Goal
Adds support for providing custom HTTP headers for every response. 

### 🧪 Manual Testing Notes
1. Open Proxyman
2. Observe Requests in the DemoApp
3. It should include a new header "Custom": "Example"

### ☑️ Contributor Checklist
